### PR TITLE
Added eth-typing from pypi

### PR DESCRIPTION
**Pull request by step21**
@conda-forge/help-python

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/eth-typing/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```
Comment (synapticarbors) [inline recipes/eth-typing/LICENSE:1]: You should remove this file as it's included in the source distribution.

**Comment by step21** [comment]
Removed License file, sry I thought I should have been added.

**Comment by synapticarbors** [comment]
Please re-merge `master` into this branch since a fix for the windows error has just been merged into `master`.

**Comment by step21** [comment]
Master is merged and win check passes.

**Pull request merged**